gradle-plugin/src/main/kotlin/dev/shreyaspatil/composeCompilerMetricsGenerator/plugin/task/ComposeCompilerReportGenerateTask.kt

```diff
@@ -156,9 +157,42 @@ abstract class ComposeCompilerReportGenerateTask : DefaultTask() {
     }
 }
 
+fun Project.registerComposeCompilerReportGenTaskForTarget(
+    target: KotlinTarget,
+    buildType: String? = null,
+): TaskProvider<ComposeCompilerReportGenerateTask> {
+    val taskName = target.name + (buildType ?: "") + "ComposeCompilerHtmlReport"
+    val compileKotlinTaskName = compileKotlinTaskNameFromTarget(target, buildType)
+    val descSuffix =
+        buildString {
+            append("'${target.name}' target")
+            if (buildType != null) {
+                append(" '$buildType' variant")
+            }
+            append(" in multiplatform project")
+        }
+    return registerComposeCompilerReportGenTask(taskName, compileKotlinTaskName, descSuffix)
+}
+
+fun Project.registerComposeCompilerReportGenTaskForJvmProject(projectName: String): TaskProvider<ComposeCompilerReportGenerateTask> {
+    val taskName = projectName + "ComposeCompilerHtmlReport"
```

Comment:
for JVM target, shouldn't we prepend "jvm"? So task would look like "jvmComposeCompilerHtmlReport"